import-export-cli/cmd/mg/list.go

```diff
@@ -0,0 +1,47 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package mg
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+)
+
+const listCmdLiteral = "list"
```

Comment:
With the revamp, the list apis cmd was deprecated and changed to get apis in apictl tool(APIM related commands). It would be better to use get apis here as well, or is there any specific reason to use list apis? @wasuradananjith @npamudika

Reply:
+1 to make this "apictl mgw get apis" to be consistent with "apictl get apis"

Reply:
I used the list to imply that it would be printing a table in the cli rather than returning a directory, file or a json object. I was hoping there would be other scenarios where the get command would be reserved for. If that's the case I'll update it to get. Thanks.